scene/gui/line_edit.cpp

```diff
@@ -950,9 +949,6 @@ void LineEdit::paste_text() {
 		append_at_cursor(paste_buffer);
 
 		if (!text_changed_dirty) {
-			if (is_inside_tree()) {
-				MessageQueue::get_singleton()->push_call(this, "_text_changed");
-			}
```

Comment:
I'm not sure about this, it seems to defeat the purpose of the surrounding `text_changed_dirty` logic here.

Reply:
I'm not entirely sure about `text_changed_dirty`

> BTW about #33417, I wonder if it's a bug or an expected behavior. If you don't get any signal when LineEdit discards your input, you're not able to react to it. Maybe there'd be a need for a `text_change_failed` signal of some sort?

That's true, it actually makes sense that you'd want some sort of signal.